h/groups/__init__.py

```diff
@@ -1,4 +1,8 @@
 # -*- coding: utf-8 -*-
 
+from h.groups import search
+
+
 def includeme(config):
     config.memex_set_groupfinder('h.groups.util.fetch_group')
+    config.memex_add_search_filter(search.GroupAuthFilter)
```

Comment:
Since this line is very important, I feel that it should be tested somehow. But there is no good way of putting fixture data into the search index for functional tests. What are our alternative ways of ensuring that we always set this filter?